### DISCUSSION TITLE
Allure report shows only last day results

### DISCUSSION THREAD

**Discussion by michalkret** [allure-framework/allure2#2228] [Questions & Support] [answered]
Hi,
I'm experimenting with Allure report as I'm looking for a tool that would better visualize my test results.
I have a repo with pytest testsuits. With allure-pytest extending my nightly runs I'm generating json reports amongst the other artifacts from these runs. I extracted allure reports from a few days executions, copied them to one folder, run `allure serve <folder name>` and... got something looking just like last night results and no trends, nor time graphs I'd expect to get.
Is there's something I'm missing in process?

**Comment by delatrie** [comment, marked as answer]
Hi!

Have you seen https://allurereport.org/docs/gettingstarted-history/ ? It describes just the process you need:

> Here is an example of how to do it, assuming that your project is configured to use allure-results and allure-report directories:
> 1. Make sure you have the previous report generated in the allure-report directory.
> 2. Remove the allure-results directory.
> 3. Run the tests.
> 4. Copy the allure-report/history subdirectory to allure-results/history.
> 5. Generate the new report.

**Reply by michalkret** [reply]
Hi Maxim, thanks I missed that part, rushing to the pytest part of the docs.
If I may bother you a bit longer, if it's possible to get more details from History panel:

![image](https://github.com/allure-framework/allure2/assets/105217427/35d5c5d7-14a5-4974-b762-f7ed9973864f)

or is it only showing  results of the tests?

**Reply by delatrie** [reply]
Glad it helped!

No, unlike the Retries tab the history doesn't contain many details. If you take a look at JSON files in the history folder, you see there isn't much information there to begin with. Just statuses, some timing information, and trends.

**Reply by islam-abdelaziz83** [reply]
> Hi!
> 
> Have you seen https://allurereport.org/docs/gettingstarted-history/ ? It describes just the process you need:
> 
> > Here is an example of how to do it, assuming that your project is configured to use allure-results and allure-report directories:
> > 
> > 1. Make sure you have the previous report generated in the allure-report directory.
> > 2. Remove the allure-results directory.
> > 3. Run the tests.
> > 4. Copy the allure-report/history subdirectory to allure-results/history.
> > 5. Generate the new report.

hello @delatrie I am having an issue that no data displayed in the history tab. and when I tried to do the steps you mentioned above I found that my project has no allure-report folder -however I can generate the allure report- my project is a Java with Maven and TestNG project and I use:- 

<plugin>
                <groupId>io.qameta.allure</groupId>
                <artifactId>allure-maven</artifactId>
                <version>2.11.2</version>
                <executions>
                    <execution>
                        <goals>
                            <goal>report</goal>
                        </goals>
                    </execution>
                </executions>
            </plugin>
            
  and: -
  
  <dependency>
            <groupId>io.qameta.allure</groupId>
            <artifactId>allure-testng</artifactId>
            <version>2.29.1</version>
        </dependency>
        
Your support is highly appreciated

**Reply by zhangxyweb** [reply]
@michalkret could you share your code for about "Copy the allure-report/history subdirectory to allure-results/history." I use the command: cp -r /tmp/allure-history-temp/allure-report/history/* allure-results/history/, but it seems not work.

**Reply by zhangxyweb** [reply]
> Glad it helped!
> 
> No, unlike the Retries tab the history doesn't contain many details. If you take a look at JSON files in the history folder, you see there isn't much information there to begin with. Just statuses, some timing information, and trends.

I use jenkins and each build i will to delete the workspace in the slave. so should i also need to remove  the allure-results directory? And also after Run the tests，the report and the allure-results/history will generated and I need to Copy the allure-report/history subdirectory to allure-results/history      
 and regenerate the new report, is my flow right?
